asserts/database.go

```diff
@@ -89,6 +89,18 @@ var (
 	ErrNotFound = errors.New("assertion not found")
 )
 
+// SupersededRevisionError is the error returned when an assertion is added
```

Comment:
I think surfacing at current could be useful, is the bit of info the caller doesn't quite have without doing extra finds/gets (revision itself is something it should have at end)